lib/content-services/src/lib/viewer/components/alfresco-viewer.component.html

```diff
@@ -24,6 +24,7 @@
     [urlFile]="urlFileContent"
     [tracks]="tracks"
     [readOnly]="readOnly"
+    [allowRotateInReadOnly]="allowRotateInReadOnly"
```

Comment:
can we be less verbose without losing meaning?
`rotateInReadOnly`
`alwaysRotate`
wdyt?

Reply:
Please check [here] (https://github.com/Alfresco/alfresco-ng2-components/pull/9873#discussion_r1657456365)